src/bloqade/submission/ir/braket.py

```diff
@@ -117,16 +185,34 @@ def from_braket_task_results(
     )
 
 
-def from_braket_status_codes(braket_message: str) -> QuEraTaskStatusCode:
-    if braket_message == str("QUEUED"):
+def from_braket_status_codes(braket_status: str) -> QuEraTaskStatusCode:
+    """Gets the `QuEraTaskStatusCode` object for working with Bloqade SDK.
+
+    Args:
+        braket_status: str
+            The value of status in metadata() in the Amazon Braket.
+            `GetQuantumTask` operation. If use_cached_value is True,
+            the value most recently returned from
+            `GetQuantumTask` operation is used
+
+    Returns:
+        An object of the type `Field` in Braket SDK
+    """
+    if braket_status == str("QUEUED"):
         return QuEraTaskStatusCode.Enqueued
     else:
-        return QuEraTaskStatusCode(braket_message.lower().capitalize())
+        return QuEraTaskStatusCode(braket_status.lower().capitalize())
+
 
+def to_quera_capabilities(paradigm) -> cp.QuEraCapabilities:
+    """Converts to `QuEraCapabilities` object supported by Braket.
 
-def to_quera_capabilities(paradigm):
-    import bloqade.submission.ir.capabilities as cp
+    Args:
+        paradigm: Bracket paradigm
```

Comment:
I think better to have @weinbe58 comment on what this variable is.

Reply:
This are the Braket version of the `QuEraCapabilities` object which is formatted slightly different than what our internal API generates. I would suggest something like:

"paradigm: The `paradigm` property of the `AwsDevice` object for Aquila" 

Mostly because the type is not really well documented or easy to find in the braket SDK.